kuma/static/styles/includes/mixins.styl

```diff
@@ -50,6 +50,13 @@ set-larger-font-size() {
     set-font-size($base-bump-font-size);
 }
 
+media-query-emify(value) {
+    /* media queries work of browser base font size (16px),
+       even if we change html font size */
+    v = unit(value / 16px, 'em');
```

Comment:
I tried it with our base font size and it put all the break points in the wrong places. The calculations work as expected with the browser's base font size of 16px.